src/Options.js

```diff
@@ -36,15 +36,15 @@ const sizeAtStyle = function(size: number, style: StyleInterface): number {
     return style.size < 2 ? size : sizeStyleMap[size - 1][style.size - 1];
 };
 
-export type OptionsData = $Shape<{
+export type OptionsData = {
     style: StyleInterface;
-    color: string;
-    size: number;
-    textSize: number;
-    phantom: boolean;
-    font: string;
+    color?: ?string;
+    size?: ?number;
+    textSize?: ?number;
+    phantom?: ?boolean;
+    font?: ?string;
```

Comment:
These only need the first `?`.  The second `?` means that they can be assigned the value `null`, but I don't think we do that anywhere.

Reply:
`color` and `font` need the second `?` in order to pass type checking in the last line in `extend`, since those properties are nullable on `Options`.